config/jobs/kubernetes/sig-cloud-provider/azure/release-master.yaml

```diff
@@ -245,6 +245,8 @@ presubmits:
         env:
         - name: KUBETEST_CONF_PATH
           value: /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/data/kubetest/conformance-fast.yaml
+        - name: CONFORMANCE_NODES
+          value: "25"
```

Comment:
what is this number based on?

Reply:
It's based on the default number of ginkgo nodes when e2e tests are run in parallel - https://github.com/kubernetes/kubernetes/blob/master/hack/ginkgo-e2e.sh#L135-L139